js/about/preferences.js

```diff
@@ -317,9 +317,26 @@ class TopBarButton extends ImmutableComponent {
   }
 }
 
+class HelpfulHints extends ImmutableComponent {
+  render () {
+    return <div className='helpfulHints'>
+      <span className='hintsTitleContainer'>
+        <span data-l10n-id='hintsTitle' />
+        <span className='hintsRefresh fa fa-refresh'
+          onClick={this.props.refreshHint} />
+      </span>
+      <div data-l10n-id={`hint${this.props.hintNumber}`} />
+      <div className='helpfulHintsBottom'>
+        <a data-l10n-id='sendUsFeedback' href={appConfig.contactUrl} />
+      </div>
+    </div>
+  }
+}
+
 class TopBar extends ImmutableComponent {
   render () {
-    return <div className='preferencesTopBar'>
+    return <div className='prefAside'>
+      <div className='prefAsideHeading'>Browser Settings</div>
```

Comment:
You can `git grep sendUsFeedback` and follow the same.  Note that you would only add it to the en_US locale and not the others. The others are auto generated.